bitops.schema.yaml

```diff
@@ -24,6 +24,10 @@ terraform:
             export_env: TERRAFORM_COMMAND
             required: True
             default: plan
+          skip-deploy:
+            type: boolean
+            parameter: skip-deploy
+            export_env: TERRAFORM_SKIP_DEPLOY
```

Comment:
Can we document it in the README as well?

Same is required for the official docs https://bitops.sh/tool-configuration/configuration-terraform/